composer.json

```diff
@@ -21,18 +21,18 @@
     "extra": {
     },
     "require": {
-        "php": "^5.6 || ^7.0",
+        "php": "^7.4",
```

Comment:
does a 7.3 drop make sense at this point? Most of the components are currently set to 7.3 and even with a component with 8.0 support, 7.3 is allowed as a minimum.

Reply:
Actually, I would prefer that you do this against the 3.4.x branch, and keep the constraints as noted here; a new minor release can then bump to 7.3 + 8.0.

This will make tests fail for the locked runs on PHP 5.6, 7.0, 7.1, and 7.2. The way to resolve that is to add a `.laminas-ci.json` file with the following contents:

```json
{
    "exclude": [
        {"name": "PHPUnit on PHP 5.6 with locked dependencies"},
        {"name": "PHPUnit on PHP 7.0 with locked dependencies"},
        {"name": "PHPUnit on PHP 7.1 with locked dependencies"},
        {"name": "PHPUnit on PHP 7.2 with locked dependencies"}
    ]
}
```

This way we know that there are no changes in functionality _before_ we do any bumps in dependency constraints.

(You'll need to add that new file to the `.gitattributes` file as well.)